src/main/groovy/io/jenkins/infra/repository_permissions_updater/ArtifactoryPermissionsUpdater.groovy

```diff
@@ -243,7 +243,7 @@ class ArtifactoryPermissionsUpdater {
                         if (!definition.cd?.exclusive) {
                             users definition.developers.collectEntries { developer ->
                                 def existsInArtifactory = KnownUsers.existsInArtifactory(developer)
-                                def existsInJira = KnownUsers.existsInJira(developer) || JiraAPI.getInstance().isUserPresent(developer)
```

Comment:
I can't see why this would be needed, we use and trust the published report just like we do for artifactory.

This code was dead because it never gets executed due to everyone being in the report

Reply:
> I can't see why this would be needed

Chesterton's fence.

https://github.com/jenkins-infra/repository-permissions-updater/pull/1856 / https://issues.jenkins.io/browse/INFRA-2835 explains it. I think that pre-dated corresponding hosting checks.

Reply:
Yes the hosting checks were added to this repository beginning of 2022 and only ever checked the report.

From what I can tell this is no longer needed, the report is now a reference and has been used as a requirement of hosting for the last couple of years without any issues.